packages/babel-preset-env/src/index.js

```diff
@@ -273,11 +286,20 @@ export default declare((api, opts) => {
     shouldTransformESM: modules !== "auto" || !api.caller?.(supportsStaticESM),
     shouldTransformDynamicImport:
       modules !== "auto" || !api.caller?.(supportsDynamicImport),
+    shouldTransformExportNamespaceFrom: !(
+      (modules === "auto" && api.caller?.(supportsExportNamespaceFrom)) ||
+      (modules === false &&
+        !isRequired("proposal-export-namespace-from", transformTargets, {
+          compatData,
+          includes: include.plugins,
+          excludes: exclude.plugins,
+        }))
+    ),
```

Comment:
Nit: This is a bit hard to follow. Could we move the `!` inside to match the same `!`/`||` pattern used in the lines above?

```suggestion
    shouldTransformExportNamespaceFrom: 
      (modules !== "auto" || !api.caller?.(supportsExportNamespaceFrom)) &&
      (modules !== false || isRequired("proposal-export-namespace-from", transformTargets, {
          compatData,
          includes: include.plugins,
          excludes: exclude.plugins,
        }))
    ),
```

Or maybe even this doesn't change the readability 🤔😅

Reply:
Well I initially unwrap `!` but these two `modules !== false || something` seems pretty mind-jiggling to me. I end up with current patterns. I can wrap these conditions into a new `shouldSkipExportNamespaceFrom` and it should be more straightforward.